docs/api/ShallowWrapper/contains.md

```diff
@@ -65,7 +65,7 @@ const wrapper = shallow((
   </div>
 ));
 
-expect(wrapper.contains(<div data-foo="foo" data-bar="bar">{4}</div>)).to.equal(true);
+expect(wrapper.contains(<div data-foo="foo" data-bar="bar">{4}</div>)).toBe(true);
```

Comment:
All assertions in our documentation use chai's expect, and intentionally do NOT use jest's jasmine-inspired assertion syntax.

This is correct as-is.